install/linux/usr/share/applications/odemis-select-mic.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Odemis with Mic Selector
+Comment=Open odemis with a microscope selector
+Keywords=odemis.conf
+Exec=gnome-terminal -- bash -c 'odemis-select-mic-start /usr/share/odemis/sim/; exec bash'
```

Comment:
You can find example of usage in the `mic-odm-yaml` repo already. It's good to have an example in the main odemis source anyway!
I think it should be `Exec=bash -c "/usr/bin/odemis-select-mic-start --select-odemis-version --select-type /usr/share/odemis/sim/*.yaml"

Reply:
I needed a terminal to spawn alongside it, since sometimes `odemis-stop` or `odemis-start` requires a sudo password. With just the bash command it didn't spawn for me.

Reply:
This it what I get
```
DEBUG:root:Running: sudo odemisd --daemonize --log-level 2 --log-target /var/log/odemis.log /usr/share/odemis/sim/meteor-zeiss-sim.odm.yaml
[sudo] password for moerkerken: 
```
From the code, the `sudo` call is defined in `odemisd/start.py`:
```
        odemisd_cmd = ["sudo", "odemisd", "--daemonize",
                 "--log-level", self._config["LOGLEVEL"],
                 "--log-target", self._config["LOGFILE"],
                 modelfile]
```

Reply:
There is a config in `sudo` so that `sudo /usr/bin/odemisd` doesn't require a password (precisely for this use case).
I think it's a sign that on your computer something is not configured according to the standard. Most likely, there is another `odemisd`, maybe from a manual installation? (see `type odemisd`).